crates/aptos/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 - Added an option `--print-metadata` to the command `aptos move download` to print out the metadata of the package to be downloaded.
   - Example: `aptos move download  --account 0x1 --package AptosFramework --url https://mainnet.aptoslabs.com/v1 --print-metadata`
 
+### Updated
+- The `--with-faucet` flag has been removed from `aptos node run-local-testnet`, we now run a faucet by default. To disable the faucet use the `--no-faucet` flag.
+- When using `aptos node run-local-testnet` we now expose a transaction stream. Learn more about the transaction stream service here: https://aptos.dev/indexer/txn-stream/. Opt out of this with `--no-txn-stream`.
```

Comment:
Please note that this is a breaking change, and note the new default ports that will be also used.

Say if someone was using this for CI, and then it starts failing because there's an extra port running.

Reply:
Good point. Odds are low this is breaking for anyone but since it could be, I'll mention it.